src/lib/components/chat/ChatWindow.svelte

```diff
@@ -325,7 +325,7 @@
 						{#if isFileUploadEnabled}
 							<UploadBtn bind:files mimeTypes={activeMimeTypes} classNames="ml-auto" />
 						{/if}
-						{#if messages && lastMessage && lastMessage.interrupted && !isReadOnly}
+						{#if messages && lastMessage && lastMessage.interrupted && !isReadOnly && !currentModel.id.includes("claude")}
```

Comment:
Something is probably wrong in the endpoint code if you have to do this, see my other comment